04.ls/ls.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+COLUMN_NUMBER = 3
+
+def main
+  directory_files = Dir.children(ARGV[0] || '.').reject { |file| file[0] == '.' }.sort
```

Comment:
```suggestion
  files = Dir.children(ARGV[0] || '.').reject { |file| file[0] == '.' }.sort
```

directoryじゃないfilesがあるわけではない（他のfilesと区別する必要がない）ので、filesだけにしてしまうのもありです。

Reply:
`files`に変更しました。